### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by YuvrajM108**
This PR is for Milestone 2, Using Bootstrap, 
The project is a clone of the Newsweek website for study purposes at Microverse. Build with:

- HTML
- CSS Framework (Bootstrap)
- CSS
- Git Workflows
This project is a collaboration between @Aduda-Boaz  and @YuvrajM108

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -115,6 +118,58 @@ button.searchButton {
...
 
+.sponsored-insight {
+  margin: 200px 10px 20px 10px;
+}
+
+.sponsored-title {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-size: 28px;
+  color: rgb(13, 93, 199);
+  font-weight: bold;
```
Comment (BigWizzo) [inline styles.css]: - [x] Remove all the unnecessarily repeated CSS rules to keep your code DRY. In this case `font-weight: bold` is used multiple times. Consider using Bootstrap class `class="font-weight-bold"`

File: styles.css [hunk 2]
```diff
@@ -115,6 +118,58 @@ button.searchButton {
...
+}
+
+.sponsored-highlight a {
+  text-decoration: none;
```
Comment (BigWizzo) [inline styles.css:123]: - [x] Remove repeated CSS rules to keep your code DRY. In this case `text-decoration: none` is used multiple times.

File: styles.css [hunk 3]
```diff
@@ -115,6 +118,58 @@ button.searchButton {
...
+  color: #171718;
+}
+
+.opinion-link h2 {
+  font-size: 20px;
+}
+
+.highlight {
+  font-size: 18px;
+  color: #6c6b6e;
```
Comment (BigWizzo) [inline styles.css:157]: - [ ] Remove repeated CSS rules to keep your code DRY. In this case `color: #6c6b6e;` is used multiple times.

**Review by BigWizzo — CHANGES_REQUESTED** [review]
# 👋 Hello @Aduda-Boaz and @YuvrajM108 

  # [STATUS] CHANGES REQUIRED♻️

Congrats on the amazing work you guys already did. We're almost done, there are some minor changes that need to be implemented to proceed:

## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (You must adjust your page to really look like the original one)⤵

- [ ] Use Bootstrap CSS classes everywhere for styling. Most of your CSS rules in your stylesheet can take advantage of Bootstrap. Please remove them and use Bootstrap instead.
- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form [better understanding Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/))
- [ ] Follow the CSS specificity rule and avoid writing code like this `a.topstory { * CSS rules here* }` but rather like this `.topstory { * CSS rules here* }`
- [ ] Below I am going to highlight some of the areas you need to adjust for your replica to look like the original.

Screenshot

- [x] Pay attention to the margins and padding for your images to align as in the original site. For example, the first images of columns 1 and 2.
- [x] Please note that some of the images in the original site are tagged e.g US, POLITICS, SPONSORED INSIGHT, RANKINGS, ETC.  Go ahead and tag yours.
- [x] Pay attention to font-size and font-weight. This will help you get a more original look.
- [ ] Under Opinions, take note of the 'VS' as well as the debate-deco.

- [ ] Please make sure that the My Turn and the Culture & Travel section look like the original. Images in the original site maintain a uniform width and height.
Screenshot (925)
- [ ] Adjust your Column 3 Sponsored insights / Rankings to look like the original
- Original
Screenshot (927)
- Yours
Screenshot (929)
- [x] Pay attention to your breaking points and fix the horizontal scroll appearing in the medium-sized devices view.
Screenshot (934)
- [ ] Adjust your My Turn and Culture & Travel for medium-sized devices to look like the original
- Original
Screenshot (935)
- Yours
Screenshot (936)

- [ ] Please make sure your breaking points match the ones on the original website.
- [ ] Most of your images in the mobile view are shrinking out of shape. You need to make them responsive and keep the shape. Take a look at the images below.
Screenshot (939)
Screenshot (940)
- ### The goal of this project is to make a Clone, hence it must look EXACTLY like the original website. Happy Coding.

## [REMINDER]
"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), now you have only 2 code reviews left for this project."

Will Nyamunokora

Feel free to contact me if you have any additional questions

- Github: [@bigwizzo](https://github.com/bigwizzo)
- Linkedin: [willnyamunokora](https://www.linkedin.com/in/willnyamunokora/)
- Slack: [@will](https://microverse-students.slack.com/team/U0162QV2TLK)

**Review by smcommits — APPROVED** [review]
# Status: Approved :+1: 
## Hello, @YuvrajM108 and @Aduda-Boaz! You guys have done a great job. Nice work implementing the changes that the previous TSE requested. This PR is now approved. You can go ahead and merge the branch. Congratulations and good luck.

**Pull request merged**